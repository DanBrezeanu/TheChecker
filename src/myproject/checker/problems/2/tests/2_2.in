802010770 63437390 777791077 124188655 645093499 532829576 637619358 955773536 0
