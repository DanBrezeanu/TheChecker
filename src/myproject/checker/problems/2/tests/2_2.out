598334126.00
